index.js

```diff
@@ -1,29 +1,52 @@
-var path = require('path');
+/* jshint node: true */
+'use strict';
 
 module.exports = {
   name: 'list-view',
-  included: function (app) {
-    this._super.included(app);
-
-    app.import('vendor/list-view.amd.js', {
-      exports: {
-        'list-view/main': ['default'],
-        'list-view/helper': ['default'],
-        'list-view/list_item_view': ['default'],
-        'list-view/list_item_view_mixin': ['default'],
-        'list-view/list_view': ['default'],
-        'list-view/list_view_helper': ['default'],
-        'list-view/list_view_mixin': ['default']
+  treeForVendor: function() {
+
+    var klassy = new this.Funnel('bower_components', {
```

Comment:
@taras  Klassy and other dependencies being included here aren't imported into target ember-cli projects, so apps fail to come up since they don't find them in bower_components. Shouldn't these be included into the target project through `addBowerPackageToProject` in a blueprint?